conf/squid/nvmeof/ceph_nvmeof_gwgroup.yaml

```diff
@@ -0,0 +1,69 @@
+globals:
+  - ceph-cluster:
+      name: ceph
+      vm-size: ci.standard.xl
+      node1:
+        role:
+          - _admin
+          - installer
+          - mon
+          - mgr
+      node2:
+        role:
+          - mon
+          - mgr
+      node3:
+        role:
+          - mon
+          - osd
+        no-of-volumes: 4
```

Comment:
Is the storage space is sufficient? if you run multiple HA scenarios.

Reply:
I felt, we need to have separate config if we want to run multiple HA scenarios with multiGWgroups